modules/apps/29-fee/keeper/escrow.go

```diff
@@ -172,9 +170,13 @@ func (k Keeper) distributeFee(ctx context.Context, receiver, refundAccAddress sd
 			return // if sending to the refund address fails, no-op
 		}
 
-		emitDistributeFeeEvent(ctx, refundAccAddress.String(), fee)
+		if err := k.emitDistributeFeeEvent(ctx, refundAccAddress.String(), fee); err != nil {
+			panic(err)
```

Comment:
panics are fine here, they won't ever happen right now or the foreseeable future and 29-fee will likely become obsolete